src/Symfony/Component/Intl/Languages.php

```diff
@@ -111,7 +111,12 @@ public static function alpha3CodeExists(string $language): bool
 
             return true;
         } catch (MissingResourceException $e) {
-            return false;
+            static $cache;
```

Comment:
I am used to having all such declarations at the top of the method. It feels weird to see it inside a catch block.

Reply:
it's a local cache :) not sure we care. (i prefer things together as much as possible)